src/components/shapeComponents/BuilderChildWrapper.vue

```diff
@@ -1,15 +1,37 @@
 <template>
-  <div class="builder-child-container" :id="id">
-    <component :is="processComponentType(shape.componentType)" :shape="shape" :mode="mode" @updateClicked="updateClicked" :value="value" :position="position" />
-    <AddDeleteButtons
-      :show="{ minus: showButtons.minus, plus: showButtons.plus }"
-      :position="position"
-      :options="nextComponentOptions"
-      @deleteClicked="deleteClicked"
-      @addNextClicked="addNextClicked"
-    />
-    <UpDownButtons :show="{ up: showButtons.up, down: showButtons.down }" :position="position" @moveUpClicked="upClicked" @moveDownClicked="downClicked" />
+  <div class="builder-child">
+    <div class="builder-child-wrapper">
+      <AddDeleteButtons
+        :show="{ minus: showButtons.minus, plus: false }"
```

Comment:
Why have 2 sets of the add delete buttons? One for minus only and one for plus only? Does the add delete buttons component need a refactor instead?